src/app/pathman.cpp

```diff
@@ -72,7 +72,28 @@ void CPathManager::AddModAutoloadDir(const std::string &modAutoloadDirPath)
 
 void CPathManager::AddMod(const std::string &modPath)
 {
-    m_mods.push_back(modPath);
+    std::string::size_type ON;
```

Comment:
I don't need to wait for Jenkins to finish running the linter to tell you that it's not going to like naming a variable with uppercase letters :P